os/net/socket/bsd_socket_api.c

```diff
@@ -132,41 +132,41 @@ int listen(int s, int backlog)
 	return lwip_listen(s, backlog);
 }
 
-int recv(int s, void *mem, size_t len, int flags)
-{
-	/* Treat as a cancellation point */
-	(void)enter_cancellation_point();
-	int result = lwip_recv(s, mem, len, flags);
-	leave_cancellation_point();
-	return result;
-}
-
-int recvfrom(int s, void *mem, size_t len, int flags, struct sockaddr *from, socklen_t *fromlen)
-{
-	/* Treat as a cancellation point */
-	(void)enter_cancellation_point();
-	int result = lwip_recvfrom(s, mem, len, flags, from, fromlen);
-	leave_cancellation_point();
-	return result;
-}
-
-int send(int s, const void *data, size_t size, int flags)
-{
-	/* Treat as a cancellation point */
-	(void)enter_cancellation_point();
-	int result = lwip_send(s, data, size, flags);
-	leave_cancellation_point();
-	return result;
-}
-
-int sendto(int s, const void *data, size_t size, int flags, const struct sockaddr *to, socklen_t tolen)
-{
-	/* Treat as a cancellation point */
-	(void)enter_cancellation_point();
-	int result = lwip_sendto(s, data, size, flags, to, tolen);
-	leave_cancellation_point();
-	return result;
-}
+//int recv(int s, void *mem, size_t len, int flags)
+//{
+//	/* Treat as a cancellation point */
+//	(void)enter_cancellation_point();
+//	int result = lwip_recv(s, mem, len, flags);
+//	leave_cancellation_point();
+//	return result;
+//}
+//
+//int recvfrom(int s, void *mem, size_t len, int flags, struct sockaddr *from, socklen_t *fromlen)
+//{
+//	/* Treat as a cancellation point */
+//	(void)enter_cancellation_point();
+//	int result = lwip_recvfrom(s, mem, len, flags, from, fromlen);
+//	leave_cancellation_point();
+//	return result;
+//}
+//
+//int send(int s, const void *data, size_t size, int flags)
+//{
+//	/* Treat as a cancellation point */
+//	(void)enter_cancellation_point();
+//	int result = lwip_send(s, data, size, flags);
+//	leave_cancellation_point();
+//	return result;
+//}
+//
+//int sendto(int s, const void *data, size_t size, int flags, const struct sockaddr *to, socklen_t tolen)
+//{
+//	/* Treat as a cancellation point */
+//	(void)enter_cancellation_point();
+//	int result = lwip_sendto(s, data, size, flags, to, tolen);
+//	leave_cancellation_point();
+//	return result;
+//}
```

Comment:
Is the annotated function not supported by esp32?

Reply:
No, these are recovered. :)